tests/integration/test_transformers.py

```diff
@@ -81,16 +85,28 @@ async def test_negated_review_of_symptoms(self):
         This is not yet 100% accurate and will take consider time to be perfect.
         Negation is not a solved problem in NLP community.
         """
-        self.assertEqual([], await list_false_positive_ss(note_negated_ros_review_of_symptoms()))
+        false_positives = await list_false_positive_ss(
+            note_negated_ros_review_of_symptoms(), url=self.url, model=self.model
+        )
+        self.assertEqual([], false_positives)
 
     async def assertPolarityCompatible(self, text: str):
         ner = await ctakesclient.client.extract(text)
         symptoms = ner.list_sign_symptom()
         polarities_ctakes = ner.list_polarity(symptoms)
-        polarities_cnlp = await ctakesclient.transformer.list_polarity(text, ner.list_spans(symptoms))
+        polarities_cnlp = await ctakesclient.transformer.list_polarity(
+            text, ner.list_spans(symptoms), url=self.url, model=self.model
+        )
 
         self.assertEqual(polarities_ctakes, polarities_cnlp, text)
 
 
+class TestTermExistsTransformer(TestNegationTransformer):
```

Comment:
I tested that this integration test I added worked, but a bunch of other integration tests failed - issues around ctakes... maybe shoddy test design or something real changed in ctakes, but I don't think we've updated that thing in forever...  Anyway, I figured I didn't need to bite that off here as well.

Reply:
Can you make an issue for that?

Reply:
Yeah great point: https://github.com/Machine-Learning-for-Medical-Language/ctakes-client-py/issues/62